### PR TITLE
Update Autodesk

### PR DESCRIPTION
**Pull request by fbernhart**
autodesk

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/autodesk.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Autodesk icon</title><path d="M14.5736 1.0206c-.097-.097-.291-.582-.971-.582H6.8988s.971.097 1.36 1.068c0 0 1.069 2.5269 2.0399 4.9558 2.8179 6.6068 7.1898 17.099 7.1898 17.099h6.5108c.097-.097-9.3267-22.443-9.4247-22.54zm-8.8407.875L.389 14.5266c-.292.875-.098 1.9439 1.457 1.9439H6.024l3.6919-8.8418C8.5499 4.714 7.5799 2.381 7.5799 2.381c-.097-.292-.389-1.069-.972-1.069-.5829 0-.7769.486-.8739.583zm-.098 15.643h-4.469c-.777 0-1.166-.486-1.166-.486.777 1.36 3.012 5.6358 3.012 5.6358.389.486.777.777 1.36.777 1.263 0 3.2069-1.263 3.2069-1.263l7.4808-4.664z"/></svg>
```
Comment (adamrusted) [inline icons/autodesk.svg:1]: Rebuilding this from source, I've got an overlap between the sections (see below). Would we be better to keep those in, or should we stick with the smoother variation of the logo @simple-icons/maintainers?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/autodesk.svg (hunk 1 above)
Comment (mondeja) [reply icons/autodesk.svg:1]: > Rebuilding this from source, I've got an overlap between the sections (see below). Would we be better to keep those in, or should we stick with the smoother variation of the logo?

I'm not sure what do you mean. For me, giving big differences that I'm seeing with the source, maybe this icon is outdated. Could you rebuild from source @fbernhart?

Comment (fbernhart) [reply icons/autodesk.svg]: I guess it actually wasn't outdated. The SVG we've used is [this one](https://damassets.autodesk.net/content/dam/autodesk/www/homepage/fy18/adsk-white.svg) from the (more or less) footer of [their website](https://www.autodesk.com/#). It's the monochrome version, which as well can be seen used here: https://brand.autodesk.com/brand-elements/autodesk-logo#acceptable-uses

The monochrome version is slightly different to the colored one as it has gaps.

image


Anyway, I've rebuilt the icon now with the monochrome version as source. @adamrusted please have a look at it. 😊

**Review by adamrusted — APPROVED** [review]
I `totally` spotted the monochrome SVG in their footer on my first glance... 🙄 

Anyways, this looks perfect! Thanks @fbernhart 🎉

**Pull request merged**